obci/interfaces/bci/abstract_classifier.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import abc, copy
+
+
+class AbstractClassifier:
+    """Base class for all classifiers."""
+    __metaclass__ = abc.ABCMeta
+
+    def __init__(self):
+        pass
+
+    @abc.abstractmethod
+    def classify(self, chunk):
+        """Compute set of classification probabilities for a given chunk of data.
```

Comment:
shouldn't it be:
def classify(self, chunk,blink) as we've been talking before?

also in master:
def _run_classification(self):
        """This method will be run as a separate thread."""
        while True:
            request = self.queue_classification.get()
            **probabilities = self.classify(self._classifier, request['chunk'], request['blink'])**